src/lib/uniforms-surfnet/src/SubscriptionField.tsx

```diff
@@ -264,7 +264,7 @@ function Subscription({
                             {tags?.includes("SP") && (
                                 <EuiButtonIcon
                                     className="show-service-port-modal-icon-button"
-                                    id={`filter-icon-${id}`}
+                                    id={`subscription-formrow-${name}-filter-icon`}
```

Comment:
Probably used in system tests.

Reply:
Reverted and solved ID problem in testcode